.github/workflows/build-ffmpeg.yml

```diff
@@ -290,11 +308,26 @@ jobs:
         working-directory: ffmpeg_sources/x265_git
         run: |
           echo "::group::configure extra flags for cross compilation"
-          extra_configure="-DCMAKE_TOOLCHAIN_FILE=build/${{ matrix.arch }}-${{ matrix.os_type }}/crosscompile.cmake"
 
-          if [[ ${{ matrix.arch }} != "aarch64" ]]; then
-            # not currently supported in `aarch64`
-            extra_configure="-DENABLE_HDR10_PLUS=1"
+          # not currently supported in `aarch64`
+          extra_configure="-DENABLE_HDR10_PLUS=1"
+          if [[ ${{ matrix.os_type }} == "linux" && ${{ matrix.arch }} == "aarch64" ]]; then
+          extra_configure="-DCMAKE_TOOLCHAIN_FILE=build/${{ matrix.arch }}-${{ matrix.os_type }}/crosscompile.cmake"
+          elif [[ ${{ matrix.os_type }} == "macos" && ${{ matrix.arch }} == "aarch64" ]]; then
+          CC="clang"
```

Comment:
these should be in a heredoc, like some of the other "extra_configure" variables... otherwise they're not passed to the cmake step

Reply:
I think, this line `extra_configure=$(cat <<VAREOF` should be before you set all these `CC`, etc. variables.